app/assets/javascripts/controllers/catalog/catalog_item_form_controller.js

```diff
@@ -123,6 +125,7 @@ ManageIQ.angular.app.controller('catalogItemFormController', ['$scope', 'catalog
     vm.catalogItemModel.provisioning_machine_credential_id = configData.provision.credential_id;
     vm.catalogItemModel.provisioning_network_credential_id = configData.provision.network_credential_id;
     vm.catalogItemModel.provisioning_cloud_credential_id = setIfDefined(configData.provision.cloud_credential_id);
+    vm.catalogItemModel.provisioning_playbook_ttl = configData.provision.playbook_ttl;
     vm.catalogItemModel.provisioning_inventory = configData.provision.hosts;
     vm.catalogItemModel.provisioning_dialog_existing = configData.provision.dialog_id ? "existing" : "create";
     vm.catalogItemModel.provisioning_dialog_id = configData.provision.dialog_id;
```

Comment:
One of the TODO refactoring items is to use `Object.assign` instead of copying the items individually.
For e.g. -
```javascript
Object.assign(vm.catalogItemModel, configData.provision);
```
(Just a FYI, and can be done later in a separate refactoring PR)